src/common/runtime/helper/utils_worker.ts

```diff
@@ -4,6 +4,9 @@ import { setDefaultRequestAdapterOptions } from '../../util/navigator_gpu.js';
 
 import { CTSOptions } from './options.js';
 
+/*
```

Comment:
Make this a doc-comment:
```suggestion
/**
```